protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -0,0 +1,185 @@
+     * The conversion of each asset price into the stable token (eg DAI) will be done off-chain
+     *
+     * @dev The asset value may be stale at any point in time, depending onthe strategy. 
+     * It may optionally implement `checkpointAssetBalances()` in order to update those balances.
+     */
+    function latestAssetBalances() public override view returns (AssetBalance[] memory assetBalances, uint256 debt) {
+        // RAMOS strategy assets = RAMOS's DAI balance + claimed AURA & BPT rewards =
+        // (bpt.balanceOf(RAMOS) / bpt.totalSupply() * Total_DAI_Balance in the LP) + 
+        // claimed AURA & BAL rewards
+
+        // get the RAMOS's DAI balance
+        AMO__IAuraStaking amoStaking = AMO__IAuraStaking(ramos.amoStaking());
+        AMO__IPoolHelper poolHelper = AMO__IPoolHelper(ramos.poolHelper());
+        IERC20 bptToken = IERC20(ramos.bptToken());
+        address stable = ramos.stable();
+
+        uint256 stableBalanceInRamos;
+        {
+            uint256 bptTotalSupply = bptToken.totalSupply();
+            uint256 bptBalanceInRamos = amoStaking.stakedBalance() + bptToken.balanceOf(address(amoStaking));
+            (, uint256 stableBalanceInLP) = poolHelper.getTempleStableBalances();
+
+            stableBalanceInRamos = stableBalanceInLP * bptBalanceInRamos / bptTotalSupply;
+        }
+
+        // get the RAMOS strategy assets
+        uint256 length = assets.length;
+        assetBalances = new AssetBalance[](length);
+
+        address asset;
+        uint256 ramosStrategyBalance;
+        for (uint256 i; i < length; ++i) {
+            asset = assets[i];
+
+            if (asset == stable) {
+                // Sum the RAMOS balance and this contract's balance of the ERC20.
+                ramosStrategyBalance = stableBalanceInRamos + IERC20(asset).balanceOf(address(this));
+            } else {
+                // Sum the `amoStaking` balance, `rewardsRecipient` balance and this contract's balance of the ERC20.
+                // But since `rewardsRecipient` address may be a hot wallet with a bunch of different things going on, we won't include that balance
+                ramosStrategyBalance = IERC20(asset).balanceOf(address(amoStaking)) + IERC20(asset).balanceOf(address(this));
+            }
+
+            assetBalances[i] = AssetBalance({
+                asset: asset,
+                balance: addManualAssetBalanceDelta(ramosStrategyBalance, asset)
+            });
+        }
+
+        debt = currentDebt();
+    }
+
+    /**
+     * @notice Borrow a fixed amount from the Treasury Reserves and add liquidity
+     * These stables are sent to the RAMOS to add liquidity and stake BPT
+     */
+    function borrowAndAddLiquidity(uint256 amount, uint256 slippageBps) external onlyOperator {
+        // Borrow the DAI and send it to RAMOS. This will also mint `dUSD` debt.
+        treasuryReservesVault.borrow(amount, address(ramos));
+        emit Borrow(amount);
```

Comment:
This event should probably be called BorrowAndAddLiquidity, in case there are other things this strategy might need to do in the future that also borrows from TRV.